docker-compose.yml

```diff
@@ -48,3 +48,13 @@ services:
     image:  bitcoinscala/tor:latest
     entrypoint: ["/tor", "--ExitRelay", "0", "--BridgeRelay",  "0", "--SOCKSPort", "0.0.0.0:9050", "--ControlPort", "0.0.0.0:9051", "--HashedControlPassword", "16:EF3A794FD6F30EF76049147EF252111809E7D51C049FEB353B547C1553"]
     restart: on-failure
+
+volumes:
+  bitcoin-s-datadir:
+    name: "bitcoin-s"
+    driver: local
+    external: false
+    driver_opts:
+      o: bind
+      type: none
+      device: ${PWD}
```

Comment:
For backwards compatability, this should be `${PWD}/data`, but i can't figure out how to do it: https://stackoverflow.com/questions/73520753/failure-to-mount-local-volume-when-the-volume-is-a-child-directory-of-pwd/73528609#73528609